### PR TITLE
Implement literal operator expression

### LINKED ISSUE #86: Implement Literal Expression Operator dsl
https://www.mongodb.com/docs/manual/reference/operator/aggregation/#literal-expression-operator

### PR DESCRIPTION
**Pull request by masonJS**
resolves #86

### CODE REVIEW

**Review by jbl428 — COMMENTED** [review]
please add a second example :)

**Comment by haruInflearn** [comment]
@jbl428  i add example for you

**Comment by jbl428** [comment]
@haruInflearn There is lint error 😓

File: example/spring-data-mongodb/src/main/kotlin/com/github/inflab/example/spring/data/mongodb/repository/expression/LiteralExpressionRepository.kt [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+    data class EditionNumberValueDto(val id: Long, val title: String, val editionNumber: Int)
+
+    /**
+     * @see <a href="https://www.mongodb.com/docs/manual/reference/operator/aggregation/literal/#treat---as-a-literal">Treat $ as a literal</a>
+     */
+    fun addEditionNumberValue(): AggregationResults<EditionNumberValueDto> {
+        val aggregation = aggregation {
+            project {
+                +Book::title.name
```
Comment (jbl428) [inline example/spring-data-mongodb/src/main/kotlin/com/github/inflab/example/spring/data/mongodb/repository/expression/LiteralExpressionRepository.kt:48]:
```suggestion
                +Book::title
```

Comment (haruInflearn) [reply example/spring-data-mongodb/src/main/kotlin/com/github/inflab/example/spring/data/mongodb/repository/expression/LiteralExpressionRepository.kt:48]: oh, sorry master. i fix it soon

Comment (haruInflearn) [reply example/spring-data-mongodb/src/main/kotlin/com/github/inflab/example/spring/data/mongodb/repository/expression/LiteralExpressionRepository.kt]: i fix it, thank review for me

**Comment by jbl428** [comment]
thank you for your contribution :)

**Review by jbl428 — APPROVED** [review]

**Pull request merged**